source/ConditionSet.cpp

```diff
@@ -272,6 +272,14 @@ void ConditionSet::Save(DataWriter &out) const
 
 
 
+void ConditionSet::MakeNever()
+{
+	// Add the equivalent "never" condition, `"'" != 0`.
+	Add("has", "'");
```

Comment:
Is there anything preventing a plugin from declaring the `'` condition as true?

Reply:
No, I don't think anything is preventing a plugin from declaring the '\'' condition to be true. This is not a new issue, this issue only moved from Mission.cpp towards here.
I already planned to solve this in a follow-up PR, and I added 2 TODO items as reminders so that it doesn't get forgotten.

Reply:
I've been thinking. According to our data format, we can't parse
> " `

as a token. It can't be quoted in quotes nor backticks, and it has a whitespace so it must be quoted.

It's a bit hacky, but this is something plugins can't possibly override. Well, unless there are some weird edge cases within our data parsing code.

Reply:
Just have a check for one specific word and don't do hacks

Reply:
> Just have a check for one specific word and don't do hacks

Or we just validate if condition-names are valid by using a regular expression when loading conditions (and their names).

Reply:
Probably not in this PR though, but the TODO item is at least created.